2주차/10989.py

```diff
@@ -0,0 +1,23 @@
+'''
+import sys
+input = sys.stdin.readline
+N=int(input())
+my_list=[]
+for i in range(N) :
+    s= int(input())
+    my_list.append(s)
+set(my_list)
+my_list.sort()
+for i in list(my_list):
+    print(i)
+'''
+## 메모리 초과
+N = int(input())
+count=[0 for _ in range(10001)]
```

Comment:
```suggestion
count=[0] * 10001
```
리스트를 모두 같은 값으로 초기화할때는 이렇게 하는게 간단하기도 하고 실행 속도도 더 빨라요